app/models/foreman_ansible/ansible_provider.rb

```diff
@@ -46,8 +46,8 @@ def rex_ssh_password(host)
           host_setting(host, 'remote_execution_ssh_password')
         end
 
-        def rex_sudo_password(host)
-          host_setting(host, 'remote_execution_sudo_password')
+        def rex_effective_user_password(host)
+          host_setting(host, 'remote_execution_effective_user_password')
```

Comment:
Doesn't this imply a migration of old settings is necessary?
We need migration to rename setting like this:
```
Setting.where(name: 'remote_execution_sudo_password').update_all(name: 'remote_execution_effective_user_password')
```

Reply:
@ezr-ondrej Migration is written in this PR - https://github.com/theforeman/foreman_remote_execution/pull/508

Reply:
Right :+1: Thanks for the pointer, missed that out. In that case no more picks.